packages/core/networks/416.json

```diff
@@ -35,6 +35,10 @@
     "contractName": "OptimisticOracle",
     "address": "0x273CAC5468b7c50B9ab2ce598693c5f441750CC4"
   },
+  {
+    "contractName": "OptimisticOracleV2",
+    "address": "0x28077B47Cd03326De7838926A63699849DD4fa87"
```

Comment:
Is it possible to get this verified here? https://explorer.sx.technology/address/0x28077B47Cd03326De7838926A63699849DD4fa87/transactions

Reply:
I have created a task for this that I will do after the audit :)